modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -247,3 +247,61 @@ PR:
       onClick?: React.MouseEventHandler<HTMLButtonElement>;
     }
     ```
+
+### Card
```

Comment:
Not sure how I missed this haha. Still needs to be added to the table of contents though

Reply:
I was working on it during your review and pushed it right before your comment about it missing.